moderation/Slowmode.go.tmpl

```diff
@@ -0,0 +1,41 @@
+{{/*
+	This commans helps using a slowmode. it deletes a user's message if their slowmode isnt over people who you want can bypass slowmode as well as set it 
+ 
+	Usage: "-slowmode on (time in seconds) / -slowmode off "
+ 
+	Recommended trigger: ".*"
+	Trigger type: Regex
+ 
+	Credits:
+	Sponge :D
+*/}}
+ 
+{{/* CONFIGURATION VALUES START */}}
+{{$bypassperms := "ManageMessages"}}
+{{$usageperms := "ManageMessages"}}
+{{/* put in the permissions you want the use to have in order to use or bypass the command */}}
+{{/* available perms: Administrator, ManageServer, ReadMessages, SendMessages, SendTTSMessages, ManageMessages, EmbedLinks, AttachFiles, ReadMessageHistory, MentionEveryone, VoiceConnect, VoiceSpeak, VoiceMuteMembers, VoiceDeafenMembers, VoiceMoveMembers, VoiceUseVAD, ManageNicknames, ManageRoles, ManageWebhooks, ManageEmojis, CreateInstantInvite, KickMembers, BanMembers, ManageChannels, AddReactions, ViewAuditLogs  */}}
+{{/* CONFIGURATION VALUES END */}}
+ 
+{{/* ACTUAL CODE */}}
+{{ dbSet 59 "bypass" $bypassperms }}
+{{ if (in (split (index (split (exec "viewperms") "\n") 2) ", ") $usageperms) }}
+{{ if $refind := reFindAllSubmatches `\A-slowmode on (\d+)` .Message.Content }}
+{{ dbSet 55 .Channel.ID (toInt (index $refind 0 1)) }}
+{{ sendMessage nil (print "done! slowmode has been set to `" (index $refind 0 1) "`") }}
+{{ else if reFindAllSubmatches `\A-slowmode off` .Message.Content }}
+{{ dbDel 55 .Channel.ID }}
+the slowmode has been removed from this channel
+{{ end }}
+{{ end }}
+{{ if (not (in (split (index (split (exec "viewperms") "\n") 2) ", ") ((dbGet 59 "bypass").Value))) }}
+{{ if (dbGet 55 .Channel.ID) }}
```

Comment:
You're using the db entry again later. To prevent fetching twice, you can use inline variable definition - `{{ if $entry := dbGet 55 .Channel.ID }}`